deals/tests.py

```diff
@@ -101,3 +108,12 @@ def test_celery_find_best_car_for_sale(create_auto_show_for_celery, create_car_i
     assert AutoShow.objects.filter(balance=250.00).exists()
     assert CarInstance.objects.order_by('price').first().price == 15.00
     assert find_best_car_for_sale.delay().get(timeout=10) == '15.00'
+
+
+@pytest.mark.django_db
+def test_authenticate_but_forbidden_request(api_client, create_user_auto_show):
+    user = User.objects.get(username="Audi AutoShow")
+    api_client.force_authenticate(user=user)
+    url = reverse('deal-list')
+    response = api_client.get(url)
+    assert response.status_code == 403
```

Comment:
use builtin status codes `from rest_framework import status`, for example: `status.HTTP_404_NOT_FOUND`
According to DRF docs:
"Using bare status codes in your responses isn't recommended. REST framework includes a set of named constants that you can use to make your code more obvious and readable."